shell/common/common_benchmarks.sh

```diff
@@ -1808,3 +1808,17 @@ get_mounts() {
   fi
   echo -e "$(echo -e "$BENCH_DEVICE_MOUNTS"|cut -d' ' -f2)"
 }
+
+# Creates a perl substitution for a ##TEMPLATE## config variable
+# Usage: create_perl_sub TEMPLATE value [TEMPLATE] [value] ...
+create_perl_template_subs() {
+  if (( $# / 2 != ($# + 1) / 2 )); then
```

Comment:
If you want to check for an even number of arguments, why not do simply

`if (( $# % 2 == 0 )); then ...`

Reply:
As for calling "exit", aloja has a function called "die" that can be used to terminate with a message that get also written to the log.